index.bs

```diff
@@ -730,6 +822,15 @@ partial interface Gamepad {
 <dfn attribute for="Gamepad" id="gamepad-getvrdisplays-attribute">displayId</dfn>
 Return the {{VRDisplay/displayId}} of the {{VRDisplay}} this {{Gamepad}} is associated with. A {{Gamepad}} is considered to be associated with a {{VRDisplay}} if it reports a pose that is in the same space as the {{VRDisplay}} pose. If the {{Gamepad}} is not associated with a {{VRDisplay}} should return 0.
 
+<dfn attribute for="Gamepad">hasPosition</dfn>
```

Comment:
I was having similar thoughts. When reviewing getPose() for instance, there is this magic between the Gamepad and the VRCoordinateSystem of the VRFrameData...